### PR TITLE
Structural Changes

### PR DESCRIPTION
**Pull request by balasankarc**
### Changes
- Change to `libindic.module` structure
- Change library name to `libindic-ngram`
- Use pbr for packaging
- Makefile oriented building
- Make tests use testtools and testrepository
- Add CI using Travis
- Add coverage generation

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = libindic-ngram
+summary = N-Gram generator for Indic Languages
+description-file =
+  README.md
+author = Santhosh Thottingal
+author-email = santhosh.thottingal@gmail.com
```
Comment (copyninja) [inline setup.cfg:7]: May be we can make this libindic developers?.. I know original code is written by @santhoshtr but there has been lot of changes contributed by many people. And we should some how acknowledge that.

**Comment by copyninja** [comment]
Can you also add tox?. It will be helpful for local running on multiple python and also on CI like CircleCI which supports this. Please raise a new PR for the tox integration

File: setup.cfg (hunk 1 above)
Comment (balasankarc) [inline setup.cfg:7]: @copyninja What about `Indic Project`? I think we can have something like `contact@indicproject.org` as the email ID also.

@anivar @stultus

Comment (copyninja) [inline setup.cfg:7]: Yeah fine with me

**Pull request merged**